configs/bioheart-test.toml

```diff
@@ -0,0 +1,16 @@
+[workflow]
+dataset = 'bioheart'
+input_datasets = ['bioheart']
+dataset_gcp_project = 'bioheart'
+sequencing_type = 'genome'
+access_level = 'test'
```

Comment:
Looks nice! Just a note that the following options are not necessary if you are running with the analysis-runner: `dataset`, `dataset_gcp_project`, `access_level`, `billing_project`. Also, `check_inputs` and `check_expected_outputs` are true by default, so you can leave them out, or keep if you want :)

Reply:
Ok excellent, thanks! That's good to know the analysis runner populates all of that data for you (I've deleted them now).